### PR TITLE
Add gtar, setup postinst scripts correctly

### PR DESCRIPTION
**Pull request by scotthain**
Resolves https://github.com/chef/omnibus-toolchain/issues/31 and adds gtar

### CODE REVIEW

**Comment by scotthain** [comment]
@chef/engineering-services

**Comment by jaymalasinha** [comment]
👍

File: package-scripts/angry-omnibus-toolchain/postinst [hunk 1]
```diff
@@ -7,7 +7,31 @@
...
+ln -sf /opt/angry-omnibus-toolchain/embedded/bin/make /opt/angry-omnibus-toolchain/bin/gmake || error_exit "Cannot link gmake to /opt/angry-omnibus-toolchain/bin"
+ln -sf /opt/angry-omnibus-toolchain/embedded/bin/make /opt/angry-omnibus-toolchain/bin/make || error_exit "Cannot link make to /opt/angry-omnibus-toolchain/bin"
+ln -sf /opt/angry-omnibus-toolchain/embedded/bin/makedepend /opt/angry-omnibus-toolchain/bin || error_exit "Cannot link makedepend to /opt/angry-omnibus-toolchain/bin"
+ln -sf /opt/angry-omnibus-toolchain/embedded/bin/patch /opt/angry-omnibus-toolchain/bin || error_exit "Cannot link patch to /opt/angry-omnibus-toolchain/bin"
+ln -sf /opt/angry-omnibus-toolchain/embedded/bin/pkg-config /opt/angry-omnibus-toolchain/bin || error_exit "Cannot link pkg-config to /opt/angry-omnibus-toolchain/bin"
+ln -sf /opt/angry-omnibus-toolchain/embedded/bin/rake /opt/angry-omnibus-toolchain/bin || error_exit "Cannot link rake to /opt/angry-omnibus-toolchain/bin"
+ln -sf /opt/angry-omnibus-toolchain/embedded/bin/ruby /opt/angry-omnibus-toolchain/bin || error_exit "Cannot link ruby to /opt/angry-omnibus-toolchain/bin"
```
Comment (schisamo) [inline package-scripts/angry-omnibus-toolchain/postinst]: Can we DRY this up a little with a for loop?

```
for name in w1 w2 ...
do command-list
done
```

**Comment by schisamo** [comment]
👍

**Pull request merged**